services/spark-master.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "spark-master",
+  "description": "Spark Master",
+  "dependencies": {
+    "conflicts": [],
+    "install": {
+      "requires": [ "base" ],
+      "uses": []
+    },
+    "provides": [],
+    "runtime": {
+      "requires": [],
+      "uses": []
+    }
+  },
+  "provisioner": {
+    "actions": {
+      "install": {
+        "type":"chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop::spark_master]"
+        }
+      },
+      "configure": {
+        "type": "chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop_wrapper::default],recipe[hadoop::spark]"
```

Comment:
looks like this should be ``hadoop::spark_master`` instead of ``hadoop::spark``, since there is config in that recipe too.